content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,215 @@
+6. **How often should Datadog run the test?** Set the interval for how often the test will run. Intervals are available between every 15 minutes to once per week. [Contact support][12] to enable additional frequencies for your test.
+7. **Alert conditions**: Set the conditions that need to be met (`minutes`, `locations`, and `retry`) for a test to retry if it has failed.
+8. **Notify your team**: Enter an message associated with the alert condition and who the alert should be sent to. Use `@` to tag people in the body of the message and select, from the dropdown list, who to send the alert to. You can also type in an email address in this location to send to any team or member that is not listed.
+9. Once you've input all required fields, click the *Save Details & Record Test* button.
+
+#### Record a test with variables and assertions
+
+Tests can be only recorded from **[Google Chrome][1]**. To record your test, download the [Datadog Record Test extension for Google Chrome][2].
+
+{{< img src="synthetics/browser_tests/browser_check_record_test.png" alt="Browser test record test" responsive="true" >}}
+
+1. Optionally, select **Open in a pop-up** at the upper right of the page to open your test recording in a separate pop-up window in order to avoid sizing issues in the displayed window within Datadog's interface.
+2. Click on **Start recording** to begin recording your browser test.
+3. Your actions are recorded and used to create steps within your browser test scenario. You can record the uploading of files as an action, though this is limited to 10 files, with a limit of 5MB each.
+4. Use the actions available in the upper left corner to enrich your scenario:
+    {{< img src="synthetics/browser_tests/browser_test_step.png" alt="Browser Test steps" responsive="true" style="width:80%;">}}
+**Note**: **Your last browser test step must be an Assertion**, otherwise there is nothing to check.
+5. Click the **Variables** button under *Add New*. Select `Pattern`, `Element`, `Global Variable`, or `Email` from the dropdown menu. For example, select `Email` and Datadog will create a special email variable that allows you to test that an email was sent by the UI and received by an email server.
+6. Click the **Assertion** button under *Add New*. Select an assertion, such as `Test that an email was received`.
+    
+    [Assertions][13] allow you to check whether an element, some content, or some text is available on the current page. You can also check whether a specific email was sent. For example, you can create an assertion that checks whether an email was sent and whether specific values (string, number, regex) are present within the email subject or body. This assertion leverages email variables.
+7. Select the subject condition and click **Use variable**. Select the email variable you created. This assigns the email variable you created to the assertion. Select the body condition and click **Use variable**. Select the email variable you created.
```

Comment:
To clarify things up, the email variable should **first** be used in a step that will send the email, for instance as an input step in a test: https://cl.ly/b71f122c8dcd
Then, users can also use variables in the email assertion step if they want to check that the body contains a specific name (for instance 'welcome Sarina').

Reply:
So it might be worth adding a step between 5 & 6 to show that the email variable should be used to trigger the email like in this https://cl.ly/b71f122c8dcd example